recipes/gridgen/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib"
+export CPPFLAGS="-I/$PREFIX/include $CPPFLAGS"
+
+pushd gridgen
+
+./configure --prefix=$PREFIX
+
+make
+make lib
+make shlib
```

Comment:
> So the library building isn't in `make all`

Last time I checked no. I have to re-visit upstream to see if there is any improvements.

> Also, I am guessing we are building static and shared libraries here. Is that correct?

Correct.